catalogue_pipeline/transformer/src/main/scala/uk/ac/wellcome/transformer/transformers/MiroTransformableTransformer.scala

```diff
@@ -400,35 +400,19 @@ class MiroTransformableTransformer
     * of (key, value) tuples.  Note: we don't use a map because keys aren't
     * guaranteed to be unique.
     */
-  def zipMiroFields(keys: Option[List[String]],
-                    values: Option[List[String]]): List[(String, String)] = {
-    (keys, values) match {
-      case (Some(k), Some(v)) => {
-        if (k.length != v.length) {
+  def zipMiroFields(keys: List[Option[String]],
```

Comment:
This is related to the change on `MiroTransformableData`:`image_award` can contain null values ie `[null, "Biomedical award"]`. While Jackson simply ignores `null`, circe needs the list to explicitly be a `List[Option[String]]` to be able to decode `null`.
Nice side effect is that the logic here becomes much simpler.